nbr-transporter.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "nbr-transporter",
+  "release": "11.2-RELEASE",
+  "artifact": "https://github.com/NAKIVO/iocage-plugin-nbr-transporter.git",
+  "properties": {
+    "allow_socket_af": "1",
+    "allow_set_hostname": "1",
+    "allow_raw_sockets": "1",
```

Comment:
Is this strictly necessary? It’s a security concern, so I’d hope the plugin could avoid doing so.

Reply:
see above.

Reply:
I don't believe that, subsonic is tomcat and it does not require any of these security overrides.

Reply:
Same concern with current diffs.  if allow_raw_sockets is for tomcat it shouldn’t be necessary. We have other tomcat plugins that do not require this lowering of security.